frontend/src/components/dashboard/tips/Tips.module.scss

```
@@ -52,6 +52,7 @@
   background-color: $color-white;
   width: $content-sm;
   max-width: calc(100% - $spacing-xl);
+  z-index: 1;
```

The fix is good, but I'd like z-indexes to always be accompanied by a comment explaining what it's trying to cover (so that we know when it can be removed/lowered or not):

```suggestion
  // Prevent the BlockLevelSlider thumb, which is also absolutely positioned,
  // from overlapping the tips area:
  z-index: 1;
```

---

Done! Thanks for always reminding me to leave comments